trailblazer/services/tower/utils.py

```diff
@@ -0,0 +1,21 @@
+from pathlib import Path
+from trailblazer.clients.tower.models import TowerTask
+from trailblazer.constants import FileFormat
+from trailblazer.io.controller import ReadFile
+from trailblazer.store.models import Analysis, Job
+
+
+def get_tower_workflow_id(analysis: Analysis) -> str:
+    file = Path(analysis.config_path)
+    content: dict = ReadFile.get_content_from_file(file_format=FileFormat.YAML, file_path=file)
+    return content.get(analysis.case_id)[-1]
+
+
+def create_job_from_dto(task: TowerTask) -> Job:
```

Comment:
docstring? I don't know what dto stands for 😅

Reply:
Renamed function in https://github.com/Clinical-Genomics/trailblazer/pull/463/commits/153c631d09c1ddc2285258574837beced185ed71